utils/update_checkout/update-checkout-config.json

```diff
@@ -149,7 +149,7 @@
                 "swift-corelibs-libdispatch": "main",
                 "swift-integration-tests": "main",
                 "swift-xcode-playground-support": "main",
-                "ninja": "v1.11.1",
+                "ninja": "v1.12.1",
```

Comment:
There's a bug in 1.12+ that results in dependencies not always rebuilding correctly. I haven't been able to fully investigate, but we should not do this until we know what is happening and have a fix.